src/pages/en/guides/content-collections.mdx

```diff
@@ -187,290 +217,211 @@ defineCollection({
+Both functions return content entries as defined by the [`CollectionEntry`](/en/reference/api-reference/#collection-entry-type) type.
 
-The filter function can also be used to query for nested directories within a collection. Since the `id` includes the full nested path, you can filter by the start of each `id` to only return items from a specific nested directory:
+#### Filtering collection queries
 
-```astro
----
+`getCollection()` takes an optional "filter" callback that allows you to filter your query based on an entry's `id`, `slug`, or `data` (frontmatter) properties.
+
+You can use this to filter by any content criteria you like. For example, you can filter by frontmatter properties like `draft` to prevent any draft blog posts from publishing to your blog:
+
+```js
+// Example: Filter content entries with `draft: true` frontmatter
 import { getCollection } from 'astro:content';
-const enDocs = await getCollection('docs', ({ id }) => {
-  // Return all entries in `src/content/docs/en/`
-  return id.startsWith('en/');
+const draftBlogEntries = await getCollection('blog', ({ data }) => {
+  return data.draft !== true;
 });
----
 ```
 
-### `getEntry()`
-
-`getEntry()` is a function that returns a specific entry in a collection by entry ID (file path relative to the collection). Both of these are required parameters.
+The filter argument also supports filtering by nested directories within a collection. Since the `id` includes the full nested path, you can filter by the start of each `id` to only return items from a specific nested directory:
 
-```astro
----
-import { getEntry } from 'astro:content';
-
-const enterprise = await getEntry('blog', 'enterprise.md');
----
+```js
+// Example: Filter entries by sub-directory in the collection
+import { getCollection } from 'astro:content';
+const englishDocsEntries = await getCollection('docs', ({ id }) => {
+  return id.startsWith('en/');
+});
 ```
 
-### Data returned from a collection query
+### Using content in Astro templates
 
-`getCollection()` and `getEntry()` will return entries that include:
- - `id` - a unique ID using the file path relative to `src/content/[collection]`.
- - `slug` - a URL-ready slug. Defaults to the `id` parsed by [github-slugger](https://github.com/Flet/github-slugger).
- - `data` - an object of frontmatter properties inferred from your collection schema. Defaults to `any` if no schema is configured.
- - `body` - a string containing the raw, uncompiled body of the Markdown or MDX document.
- - `render()` - a function that returns the compiled body of the Markdown or MDX document via the `<Content />` component ([See complete documentation](#rendering-entry-content)).
+Once you have queried your collection entries, access each entry directly inside of your Astro component template. This lets you to render HTML for things like links to your content (using the content `slug`) or information about your content (using the `data` property).
```

Comment:
```suggestion
Once you have queried your collection entries, you can access each entry directly inside of your Astro component template. This lets you to render HTML for things like links to your content (using the content `slug`) or information about your content (using the `data` property).
```
Not wrong! But feels like this fits the sentence, so I just wanted to make sure this was a deliberate omission. Ignore if intended!